plugins-compat-tester-cli/src/main/java/org/jenkins/tools/test/CliOptions.java

```diff
@@ -127,6 +132,10 @@
     @Parameter(names="-help", description = "Print this help message")
     private boolean printHelp;
 
+    @Parameter(names = "-overridePlugins", description = "List of plugins to use to test a plugin in place of the normal dependencies." +
+          "Format: 'PLUGIN_NAME=PLUGIN_VERSION", converter = PluginConverter.class, validateWith = PluginValidator.class)
```

Comment:
How does one pass a list? With system delimiters?
Maybe makes sense to also add it to the Java 11 demo in the makefile

Reply:
From the `JCommander` doc, `java.util.List` means you can set multiple times `-overridePlugins` options on the command line. So it would be `java [...] -overridePlugins plugin1=1.0 -overridePlugins plugin2=1.1`